### PR TITLE
Adding the Property (DisableBGPRoutePropagation) in a RouteGroup

### PR DESCRIPTION
**Pull request by neaggarwMS**
## Description

Added a new property (DisableBGPRoutePropagation) in RouteGroup
Also recorded the new test with API version 2017-11-01.

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#making-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Network/ChangeLog.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
     Changes for the current release should go under the section titled "Current Release", and should adhere to the following format:
 
     ## Current Release
+    * Add DisableBgpRoutePropagation property in RouteTable and effectiveRoute.
```
Comment (maddieclayton) [inline src/ResourceManager/Network/ChangeLog.md]: This should be added to the Current Release header (this is currently in the comments above and will not be included).

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmRouteTable.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
-Module Name: AzureRM.Network
+Module Name: Microsoft.Azure.Commands.Network.dll-Help
```
Comment (maddieclayton) [inline src/ResourceManager/Network/Commands.Network/help/New-AzureRmRouteTable.md]: Please revert this change

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmRouteTable.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Creates a route table.
 
 ```
 New-AzureRmRouteTable -ResourceGroupName <String> -Name <String> -Location <String> [-Tag <Hashtable>]
- [-Route <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSRoute]>] [-Force]
+ [-Route <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSRoute]>] [-DisableBgpRoutePropagation <Boolean>] [-Force]
```
Comment (maddieclayton) [inline src/ResourceManager/Network/Commands.Network/help/New-AzureRmRouteTable.md]: This shouldn't be formatted as a boolean - it should be just [-DisableBgpRoutePropagation].  Can you regenerate this file and see if this issue still exists?

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmRouteTable.md [hunk 4]
```diff
@@ -90,6 +90,19 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableBgpRoutePropagation
+Disable BGP Route auto propagation.```yaml
```
Comment (maddieclayton) [inline src/ResourceManager/Network/Commands.Network/help/New-AzureRmRouteTable.md]: Please move the ```yaml tag down to a new line (this is a known issue in platyPS).  This type should also be "SwitchParameter", this should be fixed when you regenerate the file.

File: src/ResourceManager/Network/ChangeLog.md (hunk 1 above)
Comment (neaggarwMS) [reply src/ResourceManager/Network/ChangeLog.md]: Sure

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmRouteTable.md [hunk 5]
```diff
@@ -90,6 +90,19 @@ Accept pipeline input: False
...
+Type: Boolean
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/New-AzureRmRouteTable.md]: @neaggarwMS this file needs to be regenerated this is of type `SwitchParameter`

**Comment by maddieclayton** [comment]
on demand run here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/483/
